packages/core/src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -229,6 +230,9 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
             this._mountComplianceSection(this._digitalRiverCheckoutData.checkoutData.sellingEntity);
 
             this._submitFormEvent = this._getDigitalRiverInitializeOptions().onSubmitForm;
+
+            const disabledPaymentMethods: string[] = features['PROJECT-4802.digital_river_paypal_support'] ? [] : ['payPal'];
```

Comment:
```suggestion
            const disabledPaymentMethods = features['PROJECT-4802.digital_river_paypal_support'] ? [] : ['payPal'];
```

Reply:
Hi @animesh1987, I do the change, could you please take another look?